earth_enterprise/src/common/khxml/khdom.h

```diff
@@ -281,10 +281,10 @@ ToElementWithChildName(khxml::DOMElement *elem,
                        const std::string &childTagName,
                        const std::vector<T> &vec)
 {
-  for (typename std::vector<T>::const_iterator iter = vec.begin();
-       iter != vec.end();
-       ++iter) {
-    AddElement(elem, childTagName, *iter);
+
```

Comment:
Minor: looks like this function could be combined with the following 3 functions using a template:
```
template <class T>
void
ToElementWithChildName(khxml::DOMElement *elem,
                       const std::string &childTagName,
                       const T &container)
{

  for (const auto& iter : container)
  {
    AddElement(elem, childTagName, iter);
  }
}
```
That's not really the point of the ticket but it's worth doing while we're cleaning things up.

Reply:
There are probably other similar functions that can be combined as well.